CIME/provenance.py

```diff
@@ -44,6 +44,7 @@ def _get_batch_job_id_for_syslog(case):
 def _extract_times(zipfiles, target_file):
 
     contents = "Target Build_time\n"
+    total_build_time = 0.0
     for zipfile in zipfiles:
         stat, output, _ = run_cmd("zgrep 'built in' {}".format(zipfile))
```

Comment:
In a similar vein, is the target build time reporting elapsed (real) or user time?

Reply:
The individual targets use the python `time` library, so should not have the issue of distinguishing between user and real times.

Reply:
Just for my reference, where is the CMAKE wrapper you use for recording this time info?

Reply:
@sarats, `cime/CIME/Tools/e3sm_compile_wrap.py` . This is the customizable target wrapper for E3SM. The default version (version the lives in the repo) just does timings.